do

```diff
@@ -24,7 +24,7 @@ usage() {
 }
 
 mydir=$(dirname "$0")
-cd "$(/bin/pwd)" && cd "$mydir" || die "can't find self in dir: $mydir"
+cd "$(pwd)" && cd "$mydir" || die "can't find self in dir: $mydir"
```

Comment:
We can't do this, as it will end up using the shell's internal pwd implementation. That has subtle differences which we are intentionally trying to bypass.

Reply:
Then we should call `pwd` as `/usr/bin/env pwd` or capture the path to `pwd` in a variable if the performance penalty is too high.

Reply:
Well, just to clarify, do you really truly have a system where the 'pwd' command is not in /bin? Because that seems like a hopelessly broken system to me. Why on earth would a system with no /bin/pwd have a /usr/bin/env?

Reply:
NixOS! NixOS has no /bin/pwd, but it maintains env and sh in typical locations for compatibility with scripts.

Reply:
Ok. We can use `env pwd` then if we must, although it seems a bit ridiculous. (There is no need for `/usr/bin/env` here; the explicit path is just to force sh to not use its internal version. If it has an internal env, that's ok.)

Reply:
Done.